docs/docusaurus/docs/core/configure_project_settings/access_secrets_managers/_aws_secrets_manager.md

```diff
@@ -0,0 +1,67 @@
+
+   ```bash title="Terminal"
+   pip install 'great_expectations[aws_secrets]'
+   ```
+
+2. Reference AWS Secrets Manager variables in `config_variables.yml`.
+
+   By default, `config_variables.yml` is located at: 'gx/uncomitted/config_variables.yml' in your File Data Context.
+
+   Values in `config_variables.yml` that start with `secret|arn:aws:secretsmanager` will be substituted with corresponding values from the AWS Secrets Manager.  However, if the keywords following `secret|arn:aws:secretsmanager` do not correspond to keywords in AWS Secrets Manager no substitution will occur.
+
+   You can reference other stored credentials within the keywords by wrapping their corresponding variable in `${` and `}`.  When multiple references are present in a value, the secrets manager substitution takes place after all other substitutions have occurred.
+
+   An entire connection string can be referenced from the secrets manager:
+
+   ```yaml title="config_variables.yml"
+    my_aws_creds:  secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|connection_string
+   ```
+
+   Or each component of the connection string can be referenced separately:
+   
+   ```yaml title="config_variables.yml"
+    drivername: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|drivername
+    host: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|host
+    port: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|port
+    username: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|username
+    password: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|password
+    database: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:dev_db_credentials|database
+    ```
+
+    Note that the last seven characters of an AWS Secrets Manager arn are automatically generated by AWS and are not mandatory to retrieve the secret. For example, the following two values retrieve the same secret:
+
+   ```yaml title="config_variables.yml"
+   secret1: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:my_secret-1zAyu6
+   secret2: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:my_secret
+   ```
+   
+3. Optional. Reference versioned secrets.
+
+   Unless otherwise specified, the latest version of the secret is returned by default. To get a specific version of the secret you want to retrieve, specify its version UUID. For example:
+
+   ```yaml title="config_variables.yml"
+   versioned_secret: secret|arn:aws:secretsmanager:${AWS_REGION}:${ACCOUNT_ID}:secret:my_secret:00000000-0000-0000-0000-000000000000
+   ```
+
+4. Optional. Retrieve specific secrets from a JSON string.
+ 
+   To retrieve a specific secret from a JSON string, include the JSON key after a pipe character `|` at the end of the secrets keywords.  For example:
```

Comment:
And this makes is clearer that  `|` is a separator for a json key.

This makes me think `connection_string` above is just another key in the json object? I still think being explicit about what is stored and the secret name would help me understand.